docs/src/guides/quarkus/deploy.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "How to Deploy Quarkus on Platform.sh"
+sidebarTitle: "Get started"
+weight: -110
+layout: single
+toc: false
+description: |
+    Create a Platform.sh account, download a few tools, and prepare to deploy Quarkus.
+---
+
+
+Quarkus is, in its own words, a cloud-native, (Linux) container-first framework for writing Java applications. It has become popular lately because of the amazingly fast boot time, incredibly low RSS memory. In this series of articles about Quarkus, we'll discuss how to deploy a Quarkus application even faster to the cloud with Platform.sh.
```

Comment:
```suggestion
Quarkus is, in its own words, a cloud-native, (Linux) container-first framework for writing Java applications. It has become popular lately because of its fast boot time and low RSS memory. In this series of articles about Quarkus, we'll discuss how to deploy a Quarkus application even faster to the cloud with Platform.sh.
```